docs/using-latest-daily.md

```diff
@@ -6,38 +6,32 @@ These instructions will get you set up with the latest build of Aspire. If you j
 
 See [machine-requirements.md](machine-requirements.md).
 
-## Add necessary NuGet feeds
-
-The latest builds are pushed to a special feed, which you need to add:
-```sh
-dotnet nuget add source --name dotnet8 https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json
-```
+## Run the workload installation script
 
-As usual this will add the feed to any existing NuGet.config in the directory or above, or else in the global NuGet.config. See [configuring NuGet behavior](https://learn.microsoft.com/en-us/nuget/consume-packages/configuring-nuget-behavior) to read more about that.
+The [workload installation script](./../eng/installLatestFromReleaseBranch.ps1)) will install the latest .NET Aspire workload from the release branch, but it can also install latest from main (latest nightly build) if the `-FromMain` flag is used (`--fromMain` on Linux/macOS).
```

Comment:
How should I choose? Something like "The main branch will generally have the most recent code, while the release branch will generally have been more tested."

Reply:
In reality, nowadays you really only want -FromMain always, as the script doesn't work with stable builds, and since all of the builds out of our release branch are stable now, it means the release branches support doesn't work now. I think the way forward is that the script should be changed such that the default (without params) installs you latest build from main, and if you specify a flag we instead install you the latest GA build.

Reply:
I can either do that in this PR or as a follow up. Since I'll be out for two weeks and some people might benefit from this today, I tempted to push support for main now and then as a follow up fix the scripts to support the above.